<launch>
  <!-- logger params-->
  <arg name="logger_logging_rate"	default="100"	doc="Ros_rate in Hz"/>
  <arg name="object_name"	default="gapter1"	doc="Object name to capture in relevant topics for logging" />
  <arg name="logging_path"	default="$(env HOME)/projects/NE_UAScodes/logs/Utku_experiments/latest/" 	doc="Path to log the measurements" />
  <arg name="vicon_server_offset"	default="8"	doc="Server timestamp offset in hours"/>

  <!-- load the logging node -->
  <node pkg="logging_measurements" type="Utku_logger_potential_landing" name="Utku_logger_potential_landing$(arg object_name)" output="screen" >
		<!-- Load parameters onto server using argument or default values above -->
		<param name="logger_logging_rate"	value="$(arg logger_logging_rate)" />
		<param name="object_name"	value="$(arg object_name)" />
		<param name="logging_path"	value="$(arg logging_path)" />
		<param name="vicon_server_offset"	value="$(arg vicon_server_offset)" />
  </node>
</launch>
